src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -977,6 +978,13 @@ private function registerSerializerConfiguration(array $config, ContainerBuilder
             $definition->addTag('serializer.normalizer', array('priority' => -900));
         }
 
+        if (class_exists('Symfony\Component\Serializer\Encoder\CsvEncoder')) {
```

Comment:
It's weird (accessing a constant on a class that doesn't exist) but I confirm it works.